psa-ext/cert-meta.cddl

```diff
@@ -0,0 +1,16 @@
+$$endorsed-value-map-extension //= (
+  2 => psa-cert-meta
+)
+
+psa-cert-meta = {
+    1  => text,   ; cert-level PSA Certification level
+    2  => text,   ; cert-num A unique number for each certificate
+    3  => text,   ; cert-issue-date Certificate date of issue
+    4  => text,   ; cert-test-lab Name of the certication lab
+    5  => text,   ; cert-holder Name of Organization which holds the Cert
+    6  => text,   ; cert-product Name of the Product been certified
+    7  => text,   ; cert-hw-version Version of HW Certified
+    8  => text,   ; cert-sw-version Version of SW certified
+    9  => text,   ; cert-type Further details on certification
+    10 => text    ; cert-dev-type Type of Cert, Chip or a Device
```

Comment:
1. are all of these mandatory?
2. the description should be moved to L199-208

BTW I'm ok with the "naked" keys for this

Reply:
> BTW I'm ok with the "naked" keys for this

Really? You'll depend on the co-located non-normative comments to make sense of the CDDL then, right? I see the "beauty" in this, but I am quite skeptical that that is a good design choice in the long run.

Reply:
Agreed and moved code points seprately. Marking this as now resovled!